keyboards/keyprez/corgi/readme.md

```diff
@@ -0,0 +1,19 @@
+# corgi
+
+![corgi](https://i.imgur.com/XctcpVr.jpeg)
+
+*Corgi is a semi modular 4 x 12 ortho keyboard. The unique thing with Corgi is that you can attach a macropad name "Flea" to use as a num/macro-pad.*
+
+* Keyboard Maintainer: [Christian Sandven](https://github.com/csandven)
+* Hardware Supported: *Pro Micro or Elite C is required*
+* Hardware Availability: *TBA*
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to jump to the bootloader?

Reply:
@drashna Not fully sure what you mean here, do you mean like flash for pro micro (:avrdude), elite c (:dfu)  or do you mean how to reset the keyboard?

Reply:
How to reset the keyboard, so you can flash it.